### ISSUE TITLE
volumegroupsnapshots not included in cluster role

### ISSUE THREAD

**Issue by Hobbit44** [piraeusdatastore/helm-charts#53]
Getting this error:

```
Failed to list v1alpha1 volumegroupsnapshots with error=volumegroupsnapshots.groupsnapshot.storage.k8s.io is forbidden: User "system:serviceaccount:system:snapshot-controller" cannot list resource "volumegroupsnapshots" in API group "groupsnapshot.storage.k8s.io" at the cluster scope
```

**Comment by Hobbit44** [comment]
Fixed by adding the following to the clusterrole:

```
- apiGroups:
  - groupsnapshot.storage.k8s.io
  resources:
  - volumegroupsnapshots
  - volumegroupsnapshotclasses
  - volumegroupsnapshotcontents
  verbs:
  - create
  - get
  - list
  - watch
  - update
  - patch
  - delete
```

**Comment by WanzenBug** [comment]
Should be fixed with fc099c9c6b4156642850f02e5600d30320ee4aae
